routes/routing-information.js

```diff
@@ -0,0 +1,102 @@
+module.exports = [
```

Comment:
@Otis0620 Nice catch to DRY these up. I wonder if it's best just to combine _routing-information.js_ and _misc.js_ into one file.

Reply:
@ybakos isn't best to have a giant structure like routing information split out into it's own file?